tests/unit/amo/components/TestPage.js

```diff
@@ -240,6 +255,466 @@ describe(__filename, () => {
     });
     render({ _log, errorHandler });
 
-    sinon.assert.calledWith(_log.warn, `Captured API Error: ${message}`);
+    expect(_log.warn).toHaveBeenCalledWith(`Captured API Error: ${message}`);
+  });
+
+  describe('Tests for GetFirefoxBanner', () => {
+    // Default props for all tests for GetFirefoxBanner, which will cause the
+    // banner to appear.
+    const props = { isAddonInstallPage: false };
+
+    describe('Not firefox', () => {
+      beforeEach(() => {
+        _dispatchClientMetadata({ userAgent: userAgents.chrome[0] });
+      });
+
+      it('renders a GetFirefoxBanner if the browser is not Firefox', () => {
+        render(props);
+
+        expect(
+          screen.getByRole('link', { name: 'download Firefox' }),
+        ).toHaveClass('GetFirefoxBanner-button');
+      });
+
+      it('renders a dismissable warning Notice', () => {
+        render(props);
+
+        expect(screen.getByClassName('GetFirefoxBanner')).toHaveClass(
+          'Notice-warning',
```

Comment:
It's a shame there isn't an `aria` property or similar we can test for (I couldn't think of what would be appropriate)

Reply:
The [`status` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role) seems like a possible candidate, but I'm not sure if it is appropriate, and still wouldn't give us granularity to differentiate between different levels (like warning, info, etc.). I think, because of the design and layout of the site, we are going to be stuck with often having to rely on classNames, which is probably okay because the impetus of this change wasn't actually to change our testing philosophy, but rather was to move off of a deprecated test framework. The philosophy shift just comes with it.